commons/src/main/java/com/expedia/www/haystack/pipes/commons/kafka/KafkaStreamStarter.java

```diff
@@ -104,38 +103,32 @@ Properties getProperties() {
     }
 
     private String getIpAnPort() {
-        final KafkaConfig kafkaConfig = getKafkaConfig();
-        return kafkaConfig.brokers() + ":" + kafkaConfig.port();
+        final KafkaConsumerConfig kafkaConfig = getKafkaConfig();
+        return kafkaConfig.getBrokers() + ":" + kafkaConfig.getPort();
     }
 
     private String getFromTopic() {
-        final KafkaConfig kafkaConfig = getKafkaConfig();
-        return kafkaConfig.fromtopic();
+        final KafkaConsumerConfig kafkaConfig = getKafkaConfig();
+        return kafkaConfig.getFromTopic();
```

Comment:
why not return getKafkaConfig().getFromTopic() ? no need to create that extra var right? same for below ones. Or, is there any particular reason?